Sources/Plasma/FeatureLib/pfPython/pyVaultAgeInfoNode.cpp

```diff
@@ -236,19 +230,16 @@ void pyVaultAgeInfoNode::SetAgeInstanceGuid( const char * sguid )
 {
 }
 
-const char * pyVaultAgeInfoNode::GetAgeDescription() const
+plString pyVaultAgeInfoNode::GetAgeDescription() const
 {
     if (fNode) {
-        char str[MAX_PATH];
-        memset(str, 0, sizeof(str));
         VaultAgeInfoNode access(fNode);
-        StrToAnsi(str, access.GetAgeDescription(), arrsize(str));
-        fAgeDescription = str;
+        return access.GetAgeDescription();
     }
-    return fAgeDescription.c_str();
+    return "";
```

Comment:
If `fAgeDescription` is not actually set or used, it should also be removed from the class declaration.